### PR TITLE
Lets nymphs pick a name after being split from a gestalt.

### PR DESCRIPTION
**Pull request by Yonnimer**
What it says in the title.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: code/modules/mob/living/carbon/alien/diona/diona_nymph.dm [hunk 1]
```diff
@@ -366,6 +366,29 @@
...
+
+
+	textbox	= "What shall we name ourself? Type in a name, or leave blank to cancel."
+
+	newname = input(src, textbox, "Choosing a name.", suggestion)
+	if (newname)
+		real_name = newname
+		name = newname
+		mind.name = newname
+		to_chat(src, "<span class=notice>We shall now be known as [real_name] !</span>")
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/diona/diona_nymph.dm]:
```suggestion
		to_chat(src, "<span class=notice>We shall now be known as [real_name]!</span>")
```

File: code/modules/mob/living/carbon/alien/diona/diona_nymph.dm [hunk 2]
```diff
@@ -366,6 +366,29 @@
...
+/mob/living/carbon/alien/proc/nymph_set_name()
+	set name = "Set Nymph Name"
+	set desc = "Choose a name for your yourself."
+	set category = "Abilities"
+
+	var/newname
+	var/suggestion = ""
+	var/textbox = ""
+	var/datum/language/L = locate(/datum/language/diona) in languages
+	if (L)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/diona/diona_nymph.dm]:
```suggestion
	if(L)
```

File: code/modules/mob/living/carbon/alien/diona/diona_nymph.dm [hunk 3]
```diff
@@ -366,6 +366,29 @@
...
+		suggestion = L.get_random_name()
+
+
+	textbox	= "What shall we name ourself? Type in a name, or leave blank to cancel."
+
+	newname = input(src, textbox, "Choosing a name.", suggestion)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/diona/diona_nymph.dm]:
```suggestion
	newname = input(src, textbox, "Choose a name.", suggestion)
```

File: code/modules/mob/living/carbon/alien/diona/diona_nymph.dm [hunk 4]
```diff
@@ -366,6 +366,29 @@
...
+	if (newname)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/diona/diona_nymph.dm]:
```suggestion
	if(newname)
```

**Comment by Yonnimer** [comment]
Closing for the time being, a new PR will be put up with the changes.

**Pull request closed without merging**